src/EventStore.Client.PersistentSubscriptions/EventStorePersistentSubscriptionsClient.RestartSubsystem.cs

```diff
@@ -0,0 +1,19 @@
+using System.Threading;
+using System.Threading.Tasks;
+
+#nullable enable
+namespace EventStore.Client {
+	partial class EventStorePersistentSubscriptionsClient {
+		/// <summary>
+		/// Restarts the persistent subscription subsystem
+		/// </summary>
+		/// <param name="userCredentials"></param>
+		/// <param name="cancellationToken"></param>
+		/// <returns></returns>
+		public async Task RestartSubsystemAsync(UserCredentials? userCredentials = null,
```

Comment:
**CONCERN:** As I recall, @jageall mentioned that we should not use gRPC implementation yet, as database changes are not ready yet.

Reply:
if you are referring to the features being missing from the server capabilities, I made a mistake. I had forgotten that we built it to automatically include new endpoints